openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py

```diff
@@ -94,17 +82,15 @@ def update_translatable_fields(cr):
                 WHERE it.type = 'model' AND it.name = %(name)s AND it.state = 'translated'
                 GROUP BY it.res_id
             )
-            UPDATE "%(table)s" m
-            SET "%(field_name)s" = CASE WHEN t.noupdate THEN m.%(field_name)s || t.value
-                                    ELSE t.value || m.%(field_name)s END
+            UPDATE {table} m
+            SET "{field}" = CASE WHEN t.noupdate IS FALSE THEN t.value || m."{field}"
```

Comment:
This switch in the condition is not correct, as you can have False or NULL, and both are interpreted as to update

Reply:
No, you are wrong. Here what we need to do is put EXACTLY the same query that odoo have in their code. I only updated the query because odoo updated their query. And to why they did that change, is explained in their PR, see:

![Selection_1761](https://github.com/OCA/OpenUpgrade/assets/25005517/80ebf3b7-d72a-426a-af68-be832db5ee83)

Reply:
We think the same both Raphael and me, hehe. I still think that there can be a possibility of having and existing `ir_model_data` record with the column set to NULL, and this will conflict with the NULL from `LEFT JOIN`, but OK if it's not a big problem in their samples.